src/vs/workbench/api/test/browser/extHostApiCommands.test.ts

```diff
@@ -81,13 +81,14 @@ suite('ExtHostLanguageFeatureCommands', function () {
 	let mainThread: MainThreadLanguageFeatures;
 	let commands: ExtHostCommands;
 	let disposables: vscode.Disposable[] = [];
+	let testConfigurationService: TestConfigurationService;
 
 	let originalErrorHandler: (e: any) => any;
 
 	suiteSetup(() => {
 		model = createTextModel(
 			[
-				'This is the first line',
+				'This is the first line   ',
```

Comment:
I'd rather modify the dummy model only for that one test specifically, like `model.setValue`

Reply:
Does the model value reset after each test?

Reply:
2a37a17641d7f7d7c18fdbd75b48afa0af6cb35a — although it's unclear to me whether this will pollute the model for subsequent tests or whether the model will reset before they run.